names/names.py

```diff
@@ -13,10 +14,19 @@
 duplicates = []  # Return the list of duplicates in this data structure
 
 # Replace the nested for loops below with your improvements
+# for name_1 in names_1:
+#     for name_2 in names_2:
+#         if name_1 == name_2:
+#             duplicates.append(name_1)
+
+lru_cache = LRUCache(10000)
+
```

Comment:
Make sure you're using a valid data structure for the project since the LRU cache relies on the dict() method.